### PR TITLE
update for latest version of graphql-voyager

### PR DESCRIPTION
**Pull request by viebel**

### CODE REVIEW

**Comment by viebel** [comment]
Please merge into master and deploy a new version of `ring-graphql-voyager` that will consume latest `graphql-voyager`

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "ring-graphql-voyager",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:viebel/ring-graphql-voyager.git",
```
Comment (vemv) [inline package.json]: These 3 LOCs are spurious

**Review by vemv — CHANGES_REQUESTED** [review]
Generally git is safer than any package manager at all, since it's immutable / content-adressed.

Why not just update the SHA the submodule is pointing at?

That sort of reasoning would be great to share in advance as an issue, as opposed to a PR that introduces more moving parts (NPM, an usual suspect in security) and various hard-to-review blobs (yarn.lock, which may theoretically sneak vulnerable dependencies).

Finally, you may want to look into https://www.webjars.org/ . Currently it doesn't host graphql-voyager but it may be a welcome contribution.

**Comment by viebel** [comment]
I tried and I was confused by the fact that `npm run build:release` reports a failure.
In fact, the files were generated under the dist folder. 
updated the PR

**Review by vemv — APPROVED** [review]
Kudos for the amend!

**Review by msprunck — APPROVED** [review]
LGTM!

**Comment by viebel** [comment]
Could you deploy a new version of the `ring-graphql-voyager` and of `ring-graphql-ui` soon?

**Comment by msprunck** [comment]
> Could you deploy a new version of the `ring-graphql-voyager` and of `ring-graphql-ui` soon?

Yes will try to do it today

UPDATE: I deployed the version 0.1.2 of `ring-graphql-ui`, could you test it?

**Pull request merged**